schemas/context/profile.schema.json

```diff
@@ -94,20 +94,23 @@
         "xdm:preferredLanguage": {
           "title": "Preferred Language",
           "type": "string",
-          "description":
-            "Preferred Language"
+          "description": "Preferred language based on ISO-639-1 format"
```

Comment:
A couple of questions:
- why not BCP 47, which is used in all other paces in XDM
- if you are using a standard, link to it using `[ISO-639-1](http://blabla)`
- provide examples, most people don't know from the top of their head what a language code looks like
- provide a pattern (if you are using BCP 47, there's one in `dc:language` to steal

Reply:
+100 - agreed on all points here.